compose/docker-compose.yml

```diff
@@ -51,7 +51,7 @@ services:
       context: ./proxy
       args:
         APP_HOST: web
-    image: openproject/proxy:${TAG:-14-slim}
+    image: openproject/proxy
```

Comment:
I also removed the OpenProject version tag here because the proxy is no longer dependent on the version of OpenProject itself.